src/index.ts

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+import initialize from '@/app';
+import { connect } from '@/connectToMongo';
+
+dotenv.config();
+
+const { MONGO_CONNECTION_STRING } = process.env as {
+  [key: string]: string;
+};
```

Comment:
there should be global process type declaration or either 

interface IProcessEnv { ... }